src/domain/consents/revoke.ts

```diff
@@ -42,13 +42,21 @@ import { Consent } from '../../model/consent'
 import { Enum } from '@mojaloop/central-services-shared'
 import SDKStandardComponents from '@mojaloop/sdk-standard-components'
 
+/**
+ * Validates if the request is initiated by a valid source
+ * Compares Consent object's initiator ID with source
+ */
 export function isConsentRequestInitiatedByValidSource (
   consent: Consent,
   request: Request): boolean {
   const fspiopSource = request.headers[Enum.Http.Headers.FSPIOP.SOURCE]
   return (consent && consent.initiatorId === fspiopSource)
 }
 
+/**
+ * Revoke status of consent object, update in the database
+ * and return consent
+ */
 export async function revokeConsentStatus (
```

Comment:
there is no need for async here

Reply:
because we're `await`ing the async update